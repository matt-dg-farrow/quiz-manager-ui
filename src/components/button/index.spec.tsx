import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import Button from ".";

describe("The Header component", () => {
  const wrapper = shallow(<Button name="Button" type="submit" />);

  it("should contain the correct number of components", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
