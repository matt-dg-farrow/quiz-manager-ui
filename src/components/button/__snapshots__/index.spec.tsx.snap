// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header component should contain the correct number of components 1`] = `
<button
  aria-label="Button form button"
  className="govuk-button"
  id="Button"
  name="Button"
  onClick=""
  type="submit"
>
  Button
</button>
`;
